lib/infoblox/resource/aaaa_record.rb

```diff
@@ -0,0 +1,9 @@
+# An AAAA (address) record maps a domain name to an IPv6 address.
+module Infoblox
+  class AAAArecord < Resource
+    remote_attr_accessor :name, :ipv6addr, :ttl,
+                         :extattr, :view, :comment, :disable, :dns_name, :use_ttl, :zone
```

Comment:
Yes you are right. extensible_attributes was there for the WAPI < 1.2 for AAAA record as well, now with the change, we can use any format we want depending on the WAPI version that we use. :+1: